src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -86,6 +86,33 @@
       },
       "required": [ "locations" ]
     },
+    "exception": {
+      "type": "object",
+      "properties": {
+        "type": {
```

Comment:
should this be 'kind' or something a bit more generic, to possibly include a signal name, for example? doesn't the type really categorize the exception?